src/components/messenger/list/conversation-list-panel.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import Tooltip from '../../tooltip';
+import { lastSeenText } from './utils';
+import { otherMembersToString } from '../../../platform-apps/channels/util';
+import { SearchConversations } from '../search-conversations';
+import { Channel } from '../../../store/channels';
+import { IconMessagePlusSquare, IconMessageQuestionSquare } from '@zero-tech/zui/icons';
+import { IconButton } from '../../icon-button';
+
+interface ConversationListPanelProperties {
```

Comment:
There's a lot in this panel that I want to refactor but this felt like an ok step to get merged in for now. The property names were just replicated from the source component, moving the tests down a level to test this component independently, etc.